include/seastar/net/tcp.hh

```diff
@@ -2086,6 +2107,31 @@ tcp_seq tcp<InetTraits>::tcb::get_isn() {
     //   ISN = M + F(localip, localport, remoteip, remoteport, secretkey)
     //   M is the 4 microsecond timer
     using namespace std::chrono;
+#ifdef SEASTAR_WITH_TLS_OSSL
+    uint32_t hash[8];
```

Comment:
Why did you choose a list of size 8?

Reply:
That's the size of a SHA256 hash.  It's twice as big as an MD5 digest